documentation/src/main/asciidoc/topics/ref_infinispan_caches.adoc

```diff
@@ -0,0 +1,31 @@
+[id='cache_configuration']
+= Cache Configuration
+You can provide cache configuration in XML or JSON format.
+
+[source,xml,options="nowrap",subs=attributes+,role="primary"]
+.XML
+----
+include::config_examples/infinispan_distributed_cache.xml[]
+----
+
+[source,json,options="nowrap",subs=attributes+,role="secondary"]
+.JSON
+----
+include::config_examples/infinispan_distributed_cache.json[]
+----
+
+[discrete]
+== XML format
+
+XML format must conform to the schema and include:
+
+* `<infinispan>` root element.
+* `<cache-container>` definition.
```

Comment:
I think Tristan changed this and it is no longer required. @tristantarrant can you confirm?

Reply:
@pruivo Weird I thought I removed this. Tristan can confirm but I'm pretty sure it's the case that fragments are supported on master and 12.0.x.